Yoast/Tests/WP/RequiredParametersUnitTest.inc

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// Missing autoload option.
+update_option( 'a', 'b' );
+
```

Comment:
Maybe also add a test case where `null` is passed as the third param ? That's the WP default value after all.

And what about when the param has been set to `false`/ `no` ?